IssueTriage.md

```diff
@@ -68,7 +68,7 @@ This is a somewhat subjective label and is interpreted in conjunction with Githu
 ## Issues ready to claim
 
 This state indicates that bugs/feature requests have reached the level of quality
-required for a contributor to begin writing code against (you can easily [filter for this list](https://github.com/meteor/meteor/labels/confirmed) by using the `confirmed` label).
+required for a contributor to begin writing code against (you can easily filter for [bugs](https://github.com/meteor/meteor/labels/confirmed) or [feature requests](https://github.com/meteor/meteor-feature-requests/labels/confirmed) that are ready to claim, by using the `confirmed` label).
```

Comment:
lol.  The ["Refined GitHub"](https://github.com/sindresorhus/refined-github) ("`rg`") browser extension that I saw @stubailo, was intrigued by, and installed.  Apparently it is changing things in the actual diff (trying to be helfpul) that it most certainly should not be!  *facepalm*

![image](https://user-images.githubusercontent.com/841294/26836474-9967cc8e-4ae3-11e7-8c6e-65a17cea2ce4.png)